README.md

```diff
@@ -44,14 +45,22 @@ vagrant ssh
 That's it!
 
 
-### Docker
+### Docker Setup
 We provide experimental support to run the development setup in a docker container using
+Please install Docker Desktop, **not** just the CLI docker engine. Vagrant does conflict with used ports from other docker containers, even if they are not in use. If you are encountering this issue during the build process you can change the `host` port in `Vagrantfile`.
 ```bash
 vagrant up --provider docker
 vagrant provision
 vagrant ssh
 ```
 
+bash in container:
+```bash
+export PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
+export PUPPETEER_SKIP_DOWNLOAD=true
+npm install typescript
+```
```

Comment:
I'd expect the downloading of chromium to be a postinstall action of the `puppeteer` npm package. Does `npm` really roll back the whole installation if a postinstall script fails? If it doesn't, an error here should affect the rest of the system.

In the best case, I'd want puppeteer to simply ignore errors during the chromium download here. If we can get that working, we can just include it in `provision_vagrant_vm.sh`. Users would not have to do anything additional then.

In docker, the headless browser is a pain anyways, at least on linux it doesn't start because sandboxing fails. So, I'd also be fine if we simply skipped chromium installation when docker is used. However, for virtualbox, we still want to provide the browser.

Btw, why wasn't this part of #1726? Was it still working back then?

Reply:
`apt-get install chromium-browser` is avaliable for arm64. But if we do install it on vagrant/docker, we do install a full chromium on every vm for no reason, takes long uses hdd space and it is useless, because as you said, it does not work the way we like. May we just skip the download on docker?

Reply:
yes, but we can workaround that by replacing the privision command for docker with `trigger.ruby do |env, machine| system("""vagrant docker-exec -it -- bash -c 'export PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true ; export PUPPETEER_SKIP_DOWNLOAD=true ; /evap/deployment/provision_vagrant_vm.sh'""") end`. Maby it is not the cleanest way, but it works and it is explicit enough to be found

Reply:
You'll still have trouble on any future `npm` operations in the container though, as the variables will not be set anymore, right?

Personally, I'd also be fine with this, but again, I think our long-term goal should be to get the frontend tests running in docker, not to further prevent that. @niklasmohrin thoughts?

(If we do this: Please only set one of the two variables, and personally I'd prefer the non-export way of specifying environment variables for a single command: `SOME_VARIABLE=true /evap/deployment/provision_vagrant_vm.sh`)

Reply:
yes, this is not a long term solution and might create future trouble.